worlds/smw/Client.py

```diff
@@ -506,7 +703,7 @@ async def game_watcher(self, ctx):
                 ctx.location_names.lookup_in_slot(item.location, item.player), recv_index, len(ctx.items_received)))
 
             if self.should_show_message(ctx, item):
-                if item.item != 0xBC0012 and item.item not in trap_rom_data:
+                if item.item != 0xBC0012 and item.item != 0xBC0015 and item.item not in trap_rom_data:
```

Comment:
bit wordy, but `item.item != trap_name_to_value[ItemName.literate_trap]` maybe? or store it in a local variable since you check against it twice.